src/styles/header.module.css

```diff
@@ -0,0 +1,21 @@
+@import url('https://fonts.googleapis.com/css?family=Lilita+One|Quicksand:300&display=swap');
+
+
+.container {   
```

Comment:
You've repeated container and innerContainer for many times, probably you can make something that's more reusable.